profiling/Cargo.toml

```diff
@@ -18,6 +18,7 @@ bitmaps = "3.2.0"
 bytes = "1.1"
 chrono = {version = "0.4", default-features = false, features = ["std", "clock", "serde"]}
 ddcommon = {path = "../ddcommon"}
+ddtelemetry = {path = "../ddtelemetry"}
```

Comment:
The crashtracker was moved to its own package, this might no longer be necessary